tests/test_mkieee.F90

```diff
@@ -9,9 +9,9 @@ program test_mkieee
   implicit none
 
 #if KIND == 4
-  integer, parameter :: num = 4
+  integer, parameter :: num = 5
   real :: rieee(num)
-  real :: a(num) = (/ 4.3000000, 5.6000000, 1.6700000, 2.3300000 /)
+  real :: a(num) = (/ 4.3000000, 5.6000000, 1.6700000, 2.3300000 -4.3000000 /)
```

Comment:
You need a comma.

Reply:
OK, there is another bug in this test.

Where it says:

`#if KIND == 4
`
Change it to:

`#ifdef KIND_4
`

Currently none of the code in the test is ever being compiled. ;-)

That's how you can forget a comma and not get a compile error.

Reply:
See #467